src/components/dishItems/components/menu/components/menuSection/MenuSection.js

```diff
@@ -0,0 +1,47 @@
+// prop-types
+import PropTypes from "prop-types";
+
+// components
+import MenuItemList from "../menuItemList";
+
+// helpers
+import { sanitizeText } from "../../../../../../utility/sanitizeText";
+import _startCase from "lodash/startCase";
+import { getSubTitle } from "./helpers/menuSection.general";
+
+// css
+import "./menuSection.css";
+
+const MenuSection = (props) => {
+  const { heading, itemList } = props;
+
+  return (
+    <div className="menu-section" id={sanitizeText(heading)}>
+      <h2>{_startCase(heading)}</h2>
+      <p className="sub-title">{getSubTitle(itemList.length)}</p>
+      <MenuItemList itemList={itemList} />
+    </div>
+  );
+};
+
+MenuSection.defaultProps = {
+  heading: "Heading",
+  itemList: {
+    itemCategory: [
```

Comment:
adding dummy item would not make sense since it will not be a real item to be added to cart. it could be empty array